src/core/login/keys.ts

```diff
@@ -64,6 +64,9 @@ export function makeKeysKit(
   login: LoginTree,
   keyInfos: EdgeWalletInfo[]
 ): LoginKit {
+  // For crash errors:
+  ai.props.log.breadcrumb('makeKeysKit', {})
```

Comment:
This seems redundant with all the other logging, but I guess we could keep it to have a trace like:

- createCurrencyWallet
- makeKeysKit (no kidding?)
- crash: Login API conflict error

Reply:
I'm erring on the side of redundancy since this could be a race-condition issue somewhere, so having more information even if it's obvious could reveal to us the race condition.